src/BasketBundle/Controller/Api/BasketController.php

```diff
@@ -355,10 +355,9 @@ public function deleteBasketBasketelementsAction($basketId, $elementId)
         }
 
         $view = \FOS\RestBundle\View\View::create($basket);
-        $serializationContext = SerializationContext::create();
-        $serializationContext->setGroups(array('sonata_api_read'));
-        $serializationContext->enableMaxDepthChecks();
-        $view->setSerializationContext($serializationContext);
+        $context = new Context();
```

Comment:
But it definitely wont work with FOSRestBundle 1.1. If this is a BC, i will target master branch.